src/plots/cartesian/axes.js

```diff
@@ -1221,7 +1221,7 @@ axes.hoverLabelText = function(ax, val) {
     var tx = axes.tickText(ax, ax.c2l(logOffScale ? -val : val), 'hover').text;
 
     if(logOffScale) {
-        return val === 0 ? '0' : '-' + tx;
+        return val === 0 ? '0' : MINUS_SIGN + tx;
```

Comment:
been around for a while, using regular dash instead of unicode minus. tested in https://github.com/plotly/plotly.js/pull/2113/commits/3aa03f7911fe6988fafcc71c1451b9c9f17bb76f#diff-846cf5b534aa0d22bdd1da2b43ac3cbaR517